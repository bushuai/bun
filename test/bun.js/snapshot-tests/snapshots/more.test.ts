describe("d0", () => {
  test("hello snapshot", () => {
    expect(1).toMatchSnapshot();
    expect("\nexport with test name noooooooo\n\n").toMatchSnapshot();
    expect(1).toMatchSnapshot();
    expect(1).toMatchSnapshot("one");
    expect(2).toMatchSnapshot();
    expect(3).toMatchSnapshot("one");
    expect("one two three").toMatchSnapshot();
    expect("\nexport[\\`hello snap'shot 2`] = `").toMatchSnapshot();
    expect("\nexport[`hello snapshot 2`] = `").toMatchSnapshot();
  });
});

describe("d0", () => {
  describe("d1", () => {
    test("t1", () => {
      expect("hello`snapshot\\").toEqual("hello`snapshot\\");
      expect("hello`snapshot\\").toMatchSnapshot();
    });
    test("t2", () => {
      expect("hey").toMatchSnapshot();
    });
  });
  test("t3", () => {
    expect("hello snapshot").toMatchSnapshot();
  });
  test("t4", () => {
    expect("hello`snapshot\\").toMatchSnapshot();
  });
});
