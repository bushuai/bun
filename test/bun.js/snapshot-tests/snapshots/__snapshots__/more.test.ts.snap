// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`d0 d1 t1 1`] = `"hello\`snapshot\\"`;

exports[`d0 d1 t2 1`] = `"hey"`;

exports[`d0 hello snapshot 1`] = `1`;

exports[`d0 hello snapshot 2`] = `
"
export with test name noooooooo

"
`;

exports[`d0 hello snapshot 3`] = `1`;

exports[`d0 hello snapshot 4`] = `2`;

exports[`d0 hello snapshot 5`] = `"one two three"`;

exports[`d0 hello snapshot 6`] = `
"
export[\\\`hello snap'shot 2\`] = \`"
`;

exports[`d0 hello snapshot 7`] = `
"
export[\`hello snapshot 2\`] = \`"
`;

exports[`d0 hello snapshot: one 1`] = `1`;

exports[`d0 hello snapshot: one 2`] = `3`;

exports[`d0 t3 1`] = `"hello snapshot"`;

exports[`d0 t4 1`] = `"hello\`snapshot\\"`;
